rust-bindings/src/lib.rs

```diff
@@ -36,6 +36,34 @@ pub fn validate_proof(
 mod tests {
     use super::*;
 
+    #[test]
+    fn test_proofs() {
+        /*
+         * Generated by printing the inputs to validate_proof in tests/test_python_bindings.py
+         * Specifically, the test_k_21 function was adapted to append 100 lines to the file.
+         */
+        let proofs = include_str!("../test_proofs.txt");
```

Comment:
not important; but I would think it would be faster to compile to read this file at run-time rather than compile time